engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/views/content-host-traces.html

```diff
@@ -16,6 +16,18 @@
     </p>
   </div>
 
+<p bst-alert="info" ng-hide="host.content_facet_attributes.katello_tracer_installed">
+  <span translate>
+    To use Tracer, install the katello-host-tools-tracer package.
+  </span>
+</p>
+
+<p bst-alert="info">
```

Comment:
For this particular banner, I feel like this banner style is too heavy for something that will be there all the time.  What do you think about `light` instead of `info` ?